code/drasil-gool/lib/GOOL/Drasil/LanguageRenderer/CommonPseudoOO.hs

```diff
@@ -474,4 +485,27 @@ docCommandInit = "- "
 docCommandSep = ": "
 authorDoc = "Authors"
 dateDoc = "Date"
-noteDoc = "Note"
\ No newline at end of file
+noteDoc = "Note"
+
+-- Python, Julia, and MATLAB --
+listSize :: (RenderSym r) => SValue r -> SValue r
+listSize l = do
+  f <- S.listSizeFunc l
+  mkVal (RC.functionType f) (RC.function f)
+
+-- Julia and MATLAB --
+listAdd :: (RenderSym r) => SValue r -> SValue r -> SValue r -> SValue r
+listAdd l i v = do
+  f <- S.listAddFunc l (S.intToIndex i) v
+  mkVal (RC.functionType f) (RC.function f)
```

Comment:
Continuing the theme of your comments, it would be great to also add to comments to these ones :smile:

Reply:
Good point 😄 I'll take a look and add comments where I can